File: app/src/main/java/com/letion/miracle/MainActivity.java
package com.letion.miracle;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.billy.cc.core.component.CC;
import com.billy.cc.core.component.CCResult;
import com.billy.cc.core.component.IComponentCallback;

/**
 * @author wuqi
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.textA).setOnClickListener(v -> {
            CC.obtainBuilder("ComponentA").build().callAsyncCallbackOnMainThread((cc, result) -> {
                System.out.println(result);
            });

        });
    }
}

File: testa/src/main/java/com/letion/testa/MainActivity.kt
package com.letion.testa

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.billy.cc.core.component.CC
import kotlinx.android.synthetic.main.testa_activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.testa_activity_main)
        testa_call_test.setOnClickListener({
            startActivity(Intent(this, TestAActivity::class.java))
        })
    }
}
